### PR TITLE
IDE-3610 UI test for available software sites

### PR DESCRIPTION
**Pull request by JoyeLuo**

### CODE REVIEW

File: ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/DialogAction.java [hunk 1]
```diff
@@ -102,6 +103,14 @@ public void addModule(String projectName) {
 
 	public class PreferencesDialogAction {
 
+		public boolean checkLiferayIdeSiteExist() {
+			openPreferenceType(INSTALL_UPDATE, AVAILABLE_SOFTWARE_SITES);
+
+			SWTBotTable table = bot.table();
```
Comment (jtydhr88) [inline ui-tests/common/com.liferay.ide.ui.liferay/src/com/liferay/ide/ui/liferay/action/DialogAction.java]: should not use bot.table() directly, please refer to another page object https://github.com/liferay/liferay-ide/blob/master/ui-tests/common/com.liferay.ide.ui.swtbot/src/com/liferay/ide/ui/swtbot/eclipse/page/ServerRuntimeEnvironmentsPreferencesDialog.java

File: ui-tests/tools/com.liferay.ide.ui.preference.tests/src/com/liferay/ide/ui/preference/tests/AvailableSoftwareSitesTests.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import org.junit.Test;
+
+/**
+ * @author Joye Luo
+ */
+public class AvailableSoftwareSitesTests extends SwtbotBase {
+
+	@Test
+	public void checkLiferayIdeSite()
+	{
```
Comment (jtydhr88) [inline ui-tests/tools/com.liferay.ide.ui.preference.tests/src/com/liferay/ide/ui/preference/tests/AvailableSoftwareSitesTests.java]: sf

**Comment by jtydhr88** [comment]
see commits, thanks.

**Comment by JoyeLuo** [comment]
Thanks, got it. Will resend

**Pull request closed without merging**